rpilocator_telegram.py

```diff
@@ -0,0 +1,78 @@
+import telegram, sys
+import feedparser
+import time
+import json
+
+# Feed URL
+FEED_URL = 'https://rpilocator.com/feed/'
+
+# Telegram settings
+TELEGRAM_BOT_TOKEN = '<your telegram bot token>'
+TELEGRAM_CHAT_ID = '<your telegram chat id>'
+
+# Customize the message title
+MESSAGE_TITLE = 'xlocator Stock Alert'
+
+# User Agent
+USER_AGENT = 'xlocator feed alert'
+
+# Create the message body
+def formatMessage(entry):
+    message = [
+        f"<b><u>{MESSAGE_TITLE}</u></b>",
+        f"",
+        f"{entry.title}",
+        f"",
+        f"{entry.link}",
+    ]
+
+    message = '\n'.join(message)
+
+    return message
+
+# Telegram Nachricht senden
+def sendMessage (message):
+    try:
+        bot = telegram.Bot(token=TELEGRAM_BOT_TOKEN)
+    except Exception as err:
+        print('Unhandled exception while creating telegram bot object: %s' % err, file = sys.stderr)
+        return False
+    try:
+        result_msg = bot.sendMessage(chat_id=TELEGRAM_CHAT_ID, text=message, parse_mode='HTML')
+        return isinstance(result_msg, telegram.message.Message)
```

Comment:
Quick question, any reason why you return anything within `sendMessage(X)` ?

Reply:
You're right. They are from another project from me. Damn copy&paste ;-)
I have removed the returns.